src/tools/dev/scripts/bv_support/bv_hdf5.sh

```diff
@@ -489,6 +489,35 @@ EOF
     return 0;
 }
 
+function apply_hdf5_resize_configs_are_equal_patch
+{
+    info "Patching hdf5 1.8.14 for isatty"
+    patch -p0 << EOF
+*** test/cache_common.h.orig	2014-10-13 04:33:37.000000000 -0700
+--- test/cache_common.h	2023-04-11 20:25:27.000000000 -0700
+*************** void check_and_validate_cache_size(hid_t
+*** 697,702 ****
+--- 697,707 ----
+                                     int32_t * cur_num_entries_ptr,
+                                     hbool_t dump_data);
+  
++ hbool_t
++ resize_configs_are_equal(const H5C_auto_size_ctl_t *a,
++     const H5C_auto_size_ctl_t *b,
++     hbool_t compare_init);
++ 
+  void validate_mdc_config(hid_t file_id,
+                           H5AC_cache_config_t * ext_config_ptr,
+                           hbool_t compare_init,
+EOF
+    if [[ $? != 0 ]] ; then
+        warn "HDF5 1.8.14 resize_configs_are_equal patch."
```

Comment:
Should the warning indicate that the patch failed?

Reply:
Yes, it should. I'll fix shortly

Reply:
```suggestion
        warn "HDF5 1.8.14 resize_configs_are_equal patch failed."
```